test/scripts/e2e-kind.sh

```diff
@@ -43,6 +40,7 @@ named port.+\[Feature:NetworkPolicy\]
 Services.+(ESIPP|cleanup finalizer)
 configMap nameserver
 ClusterDns \[Feature:Example\]
+should set default value on new IngressClass
```

Comment:
boo. Add a comment and hopefully a bug link.

otherwise lgtm

Reply:
I'm going to go through all of these ??? and figure out why each one fails and update that all at once. Either today or tomorrow (scout's honor), but let's not hold up fixing CI to root cause that one IMO. OK with you?